src/components/Explorer/QuestionnairesTableRow.vue

```diff
@@ -28,70 +24,45 @@
 </template>
 
 <script lang="ts">
-import { defineComponent, reactive } from 'vue'
+import { defineComponent } from 'vue'
 import Icon from '/@/components/UI/Icon.vue'
 
 export default defineComponent({
   name: 'QuestionnaireTableRow',
   components: {
     Icon
   },
-  setup() {
-    const data = reactive({
-      questionnaires: [
-        {
-          questionnaireID: 1,
-          title: 'こんにちは',
-          res_time_limit: '2020-12-31',
-          created_at: '2020-1-1',
-          modified_at: '2020-1-1',
-          res_shared_to: 'public',
-          is_targeted: true
-        },
-        {
-          questionnaireID: 2,
-          title: 'こんばんは',
-          res_time_limit: '2020-12-31',
-          created_at: '2020-1-1',
-          modified_at: '2020-1-1',
-          res_shared_to: 'public',
-          is_targeted: true
-        }
-      ]
-    })
-
-    return {
-      data
+  props: {
+    questionnaire: {
+      type: String,
```

Comment:
個々の型、Stringじゃないと思う
`vue`から`PropType`を、`/@/lib/apis`から`Questionnaire`をimportしてサジェストみたいにしたら補完がつく
```suggestion
      type: Object as PropType<Questionnaire>,
```